README.md

```diff
@@ -1,7 +1,91 @@
 # data-worker
+
 data-worker is a application for working with data from TMDb and putting it to database.
+
+## Getting Started
+
+These instructions will get you a copy of the project up and running on your local machine for development and testing purposes.
+
+### Prerequisites
+
+Download Node.js not lower than version 8.10, [Yarn](https://yarnpkg.com/) , [Docker](https://www.docker.com/) and [MongoDB Docker image](https://hub.docker.com/_/mongo).
+
+### Clone
+
+Clone this repo to your local machine using `https://github.com/IlonaMenkui/data-worker.git`
+
+### Run database
+<a name="run-db"></a>
+
+Run your docker image with MongoDb.
+```
+docker run --name some-mongo -d mongo:tag
+```
+... where ```some-mongo``` is the name you want to assign to your container and ```tag``` is the tag specifying the MongoDB version you want.
+
+... or run the ```make``` utility in the project directory.
+```
+make run-mongodb
+```
+
+### Declare environment variables in .env file
+
+Create and add application configuration to .env file in the root of the project:
+
+```
+APP_PORT=3000
+TMDB_API_KEY=YOURSECRETAPIKEY
+DB_URI=mongodb://localhost:27017/super-enigma-db
+TMDB_BASE_URL=https://api.themoviedb.org/3/
+```
+
+You should to register for an API key, click the [API link](https://www.themoviedb.org/settings/api) from within your account settings page.
+
 ## Installation
-Use the package manager [Yarn](https://yarnpkg.com/) to install all the dependencies of data-worker.
+
+Use the package manager Yarn to install all the dependencies of data-worker.
+
+```
+yarn install
+```
+
+## Available Scripts
+
+In the project directory, you can run:
+
+```
+yarn start
+```
+
+Runs the app.<br>
+
+The app will reload if you make changes to the code.<br>
+
+## Run Docker container with project and MongoDB
+First, make sure that [MongoDB is running](#run-db).
+Then you need to build a server.
+```
+make build-server
+```
+
+In the end, run the docker container with server.
+```
+docker run -P --env APP_PORT=app-port --env DB_URI=db-uri --name server-name --link db-name image-uri/module-name
```

Comment:
if you define env variables in Dockerfile then you don't need `--env` flags here